### PR TITLE
Test on Python 3.7

### LINKED ISSUE #22: Make travis testing for python 3.7
* Pick n Mix version: 0.1.1
* Python version: 3.7
* Operating System: Any

### Description

Right now travis is not testing for python3.7. It needed to be added.

### PR DESCRIPTION
**Pull request by hugovk**
Fixes #22.

### CODE REVIEW

**Comment by hugovk** [comment]
Looks like Travis CI isn't yet enabled for this repo.

Here's how it looks on my fork:

* https://travis-ci.org/hugovk/picknmix/builds/559364273

**Comment by Cheukting** [comment]
Yup I remove 3.7 as it was not working before for some strange reasons.

File: CONTRIBUTING.rst [hunk 1]
```diff
@@ -102,7 +102,7 @@ Before you submit a pull request, check that it meets these guidelines:
 2. If the pull request adds functionality, the docs should be updated. Put
    your new functionality into a function with a docstring, and add the
    feature to the list in README.rst.
-3. The pull request should work for Python 2.7, 3.4, 3.5 and 3.6, and for PyPy. Check
+3. The pull request should work for Python 2.7, 3.5+ and for PyPy. Check
```
Comment (Cheukting) [inline CONTRIBUTING.rst]: Mind removing 2.7 for me? This project should only support 3.5+

**Comment by Cheukting** [comment]
Thanks! Mind putting your name under contributors in AUTHORS.rst? You can skip the email if you want.

**Comment by hugovk** [comment]
Updated!

Yeah, the gotcha is Python 3.7 is only available on with Ubuntu Xenial on Travis CI (see https://github.com/travis-ci/travis-ci/issues/9815, but no longer needs `sudo`).

I also ran the [pyupgrade](https://github.com/asottile/pyupgrade) tool to automatically update some syntax for Python 3+:

```sh
pip install -U pyupgrade
pyupgrade `find . -name "*.py" -type f` --py3-plus
```

And removed the dot in `py.test` as recommended by https://twitter.com/pytestdotorg/status/753767547866972160.

**Comment by Cheukting** [comment]
All looks good! Thank you so much. I learnt a lot. :-)

**Comment by hugovk** [comment]
You're welcome!

**Pull request merged**